Automattic/wp-calypso
# Add upgrade paths from growth to complete

## Proposed Changes

* Add upgrade paths from Growth to Complete

## Why are these changes being made?

We want users to be able to upgrade from Growth to Complete from the `/me/purchases` page

## Testing Instructions

1. Checkout this diff on your sandbox: D167039-code
2. Sandbox public-api.wordpress.com
3. Purchase a Growth plan from https://wordpress.com/checkout/jetpack/jetpack_growth_yearly
4. Checkout this branch or use the Calypso live link and go to `/me/purchases`
5. Select your Growth plan
![image](https://github.com/user-attachments/assets/12781082-15ec-4178-8860-b5d489d71d50)
6. Click the Upgrade CTA and ensure you see Complete in the cart
![image](https://github.com/user-attachments/assets/cf8dd337-34ef-44dc-9f98-06e385340091)

## Pre-merge Checklist
